Build/GetWix.ps1

```diff
@@ -0,0 +1,21 @@
+function GetWix {

```

Comment:
I would actually expect the most portable way to extract a zip in PS to be ZipArchive and friends. I think we might have not used it originally because it's .NET 4.5+ only?